framework/src/media/audio/audio_manager.c

```diff
@@ -1251,6 +1418,34 @@ uint32_t get_output_bytes_to_frame(unsigned int bytes)
 	return pcm_bytes_to_frames(g_audio_out_cards[g_actual_audio_out_card_id].pcm, bytes);
 }
 
+unsigned int get_user_output_frames_to_byte(unsigned int frames)
+{
+	int buffer_size = 0;
+	audio_card_info_t *card;
+
+	if ((g_actual_audio_out_card_id < 0) || (frames == 0)) {
+		return 0;
+	}
+	card = &g_audio_out_cards[g_actual_audio_out_card_id];
+	buffer_size = frames * card->resample.user_channel * card->resample.format;
+
+	return buffer_size;
+}
+
+unsigned int get_user_output_bytes_to_frame(unsigned int bytes)
+{
+	int frame_size = 0;
```

Comment:
yes. like this. please think about name of api =)

Reply:
Okay~.